hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -174,13 +175,21 @@ public void handleRound(
         for (final var event : round) {
             final var creator = networkInfo.nodeInfo(event.getCreatorId().id());
             if (creator == null) {
-                // We were given an event for a node that *does not exist in the address book*. This will be logged as
-                // a warning, as this should never happen, and we will skip the event. The platform should guarantee
-                // that we never receive an event that isn't associated with the address book, and every node in the
-                // address book must have an account ID, since you cannot delete an account belonging to a node, and
-                // you cannot change the address book non-deterministically.
-                logger.warn("Received event from node {} which is not in the address book", event.getCreatorId());
-                return;
+                if (!isSoOrdered(event.getSoftwareVersion(), version)) {
```

Comment:
Not from the designated cherry pick - #14608 - here.  Seems to have picked up another change in develop (apparently also needed in 0.52) - L178-191.